src/components/general/modals/prompt-login-modal.js

```diff
@@ -4,15 +4,15 @@ import styled from 'styled-components';
 
```

Comment:
you probably also want to rename this file something more generic like form-modal.js

Reply:
Changed modal name to form-modal.js